### PR TITLE
Basic schema for websites yml

### PR DESCRIPTION
**Pull request by stephengroat**

### CODE REVIEW

**Comment by mxxcon** [comment]
yaml schema! 🙌 😄

**Comment by stephengroat** [comment]
i wouldn't say it's great, but it's a start. `kwalify` is a mess, but it's something

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ language: ruby
 cache: bundler
 rvm:
     - 2.4.0
+before_install:
+  - gem update --system --no-doc
```
Comment (Carlgo11) [inline .travis.yml:7]: why is this necessary?

**Review by Carlgo11 — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (stephengroat) [reply .travis.yml:7]: `gem` has a bug that's creating a frozen sting issue, fixed in v2.6.9

**Review by jamcat22 — APPROVED** [review]
